debugger/src/main.rs

```diff
@@ -337,3 +366,23 @@ fn main() -> rustyline::Result<()> {
     }
     Ok(())
 }
+
+fn check_for_updates(client: Client) -> Option<String> {
+    let response = client
+        .get("https://crates.io/api/v1/crates/pest_debugger")
+        .send();
+
+    if let Ok(response) = response {
+        response.json::<serde_json::Value>().ok().and_then(|json| {
+            let version = json["crate"]["max_version"].as_str()?;
```

Comment:
I guess this may panic if the object didn't happen to contain those... it could be made more robust with checked value getting, but probably ok as it is